pwndbg/heap/ptmalloc.py

```diff
@@ -634,7 +635,13 @@ def malloc_par(self):
 
     def get_heap(self, addr):
         """Find & read the heap_info struct belonging to the chunk at 'addr'."""
```

Comment:
Can we extend the docs here to mention why we do .fetch_lazy / what's the potential issue here?